data/simple-icons.json

```
@@ -13947,6 +13947,17 @@
 		"hex": "1D1D1D",
 		"source": "https://rive.app"
 	},
+	{
+		"title": "RKE2",
+		"hex": "2453FF",
```

```suggestion
		"hex": "1167F1",
```

From the logo graphic